src/main/resources/io/github/dfdlschemas/envelopepayload/xsd/envelopePayload.dfdl.xsd

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<schema
+  xmlns="http://www.w3.org/2001/XMLSchema"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:ep="urn:io.github.dfdlschemas.envelopepayload"
+  targetNamespace="urn:io.github.dfdlschemas.envelopepayload"
+  xmlns:msHdr="urn:milstd2045DFDL"
+  xmlns:pcap="urn:pcap:2.4"
+  elementFormDefault="unqualified">
```

Comment:
Ditto here about why elementFormDefault is here and sorting these lines alphabetically.